### PR TITLE
Issue #3040391 by FeyP, Joseph Zhao, jefftrnr, joelpittet, jzasnake, …

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop/pull/2581

### CODE REVIEW

**Comment by herbdool** [comment]
@jenlampton the new test included in this doesn't run at all. I tried to run it manually in the test environment. Makes sense since it should produce some fatal errors (e.g. filter_format is no longer a db table).

**Comment by jenlampton** [comment]
Test aside, the fix here also didn't work for me.

File: core/modules/views/tests/views.tests.info [hunk 1]
```diff
@@ -190,6 +190,12 @@ description = Tests the core views_handler_filter_string handler.
 group = Views Handlers
 file = handlers/views_handler_filter_string.test
 
+[ViewsHandlerFilterStringTest]
```
Comment (quicksketch) [inline core/modules/views/tests/views.tests.info:193]:
```suggestion
[ViewsHandlerManyToOneTest]
```

File: core/modules/views/tests/handlers/views_handler_manytoone.test [hunk 2]
```diff
@@ -0,0 +1,614 @@
...
+   * Instances of fields used by this test.
+   *
+   * @var array
+   */
+  protected $instances;
+
+  /**
+   * Nodes used by this test.
+   *
+   * @var stdClass[]
```
Comment (quicksketch) [inline core/modules/views/tests/handlers/views_handler_manytoone.test:25]:
```suggestion
   * @var Node[]
```

File: core/modules/views/tests/handlers/views_handler_manytoone.test [hunk 3]
```diff
@@ -0,0 +1,614 @@
...
+   */
+  protected $nodes;
+
+  /**
+   * Accounts used by this test.
+   *
+   * @var stdClass[]
```
Comment (quicksketch) [inline core/modules/views/tests/handlers/views_handler_manytoone.test:32]:
```suggestion
   * @var User[]
```

File: core/modules/views/tests/handlers/views_handler_manytoone.test [hunk 4]
```diff
@@ -0,0 +1,614 @@
...
+   */
+  protected $accounts;
+
+  /**
+   * Terms used by this test.
+   *
+   * @var stdClass[]
```
Comment (quicksketch) [inline core/modules/views/tests/handlers/views_handler_manytoone.test:39]:
```suggestion
   * @var TaxonomyTerm[]
```

**Comment by quicksketch** [comment]
The new tests here would be great to include, code fixes aside, there is a lot of new coverage added.

**Comment by jenlampton** [comment]
Closing in favor of https://github.com/backdrop/backdrop/pull/2581

**Pull request closed without merging**